test/spec/features/stack/validate_spec.rb

```diff
@@ -3,9 +3,21 @@
     context 'entrypoint' do
       it 'sets stack entrypoint' do
         with_fixture_dir("stack/keywords") do
-          k = run 'kontena stack validate entrypoint.yml'
-          expect(k.code).to be_zero
-          expect(k.out).to match(/entrypoint.+?foo.+?entrypoint.+?foo2/m)
+          k = run! 'kontena stack validate --format=api-json entrypoint.yml'
+          data = JSON.load(k.out)
+
+          expect(data).to match hash_including(
+            'services' => a_collection_containing_exactly(
+              hash_including(
+                'name' => 'redis',
+                'entrypoint' => 'foo',
```

Comment:
Trailing comma

Reply:
Not a problem, a matter of personal preference. Approved already with or without the commas.

There are upsides are simpler diffs in some cases and easier line moving, downside is mostly that it looks weird. 

And if you somehow manage to omit the closing bracket you may get something weird instead of an error.

Reply:
Marking as `status/merge`, because I'm pretty sure there's already plenty of trailing commas around the codebase... it doesn't sound like a code style thing that we've been using so far.

Reply:
(in 148 files to be exact, a couple of occurrences even by me)